numpy/ma/core.py

```diff
@@ -8264,8 +8283,8 @@ def _convolve_or_correlate(f, a, v, mode, propagate_mask):
     if propagate_mask:
         # results which are contributed to by either item in any pair being invalid
         mask = (
-            f(getmaskarray(a), np.ones(np.shape(v), dtype=bool), mode=mode)
-          | f(np.ones(np.shape(a), dtype=bool), getmaskarray(v), mode=mode)
+            f(getmaskarray(a), np.ones(np.shape(v), dtype=bool), mode=mode) |
```

Comment:
IIRC, Python changed the style to prefer putting the operator in front. Don't recall if that was for Python or C code.

Reply:
Indeed, see [W503](https://www.flake8rules.com/rules/W503.html) / [W504](https://www.flake8rules.com/rules/W504.html).

To be honest, I don't care about formatting, just about silencing pycodestyle. I'd rather let a formatter do the formatting for me. However, I understand black or ruff don't work well on numerical code.

Reply:
I'm happy with leaving the change here, just reminded of the preference change.